src/test/java/org/commcare/formplayer/tests/CaseClaimTests.java

```diff
@@ -120,9 +123,23 @@ public void testQueryScreen() throws Exception {
                 true,
                 QueryResponseBean.class);
         assert queryResponseBean.getDisplays()[0].getValue().contentEquals("Burt");
+        assert queryResponseBean.getDisplays()[1].getValue().contentEquals("0");
         assertArrayEquals(queryResponseBean.getDisplays()[1].getItemsetChoices(), new String[]{"karnataka", "Raj as than"});
+
+        // check if we have districts corresponding to karnataka state
         assertArrayEquals(queryResponseBean.getDisplays()[2].getItemsetChoices(), new String[]{"Bangalore", "Hampi"});
 
+
+        // multi-select test
+        inputs.put("district", "0#,#1"); // select 2 districts
```

Comment:
do you need to call `queryData.setInputs("search_command.m1", inputs);` again after modifying `inputs`?

Reply:
Not really, modifying `inputs` modify the memory reference for `queryData`. That's how we are seeing the two district values added to the search url at L185 below.

Reply:
:+1: just checking since it was called like that above